profiles/common/modules/features/ec_embedded_video/ec_embedded_video.install

```diff
@@ -4,6 +4,8 @@
  * Install, update and uninstall functions for the ec_embedded_video module.
  */
 
+module_load_include('inc', 'nexteuropa_multilingual', 'includes/nexteuropa_multilingual.helpers');
```

Comment:
You are not including the right file i think
ec_embedded_video/includes/ec_embedded_video.helpers.inc